src/main/java/ru/sbt/mipt/oop/general/Application.java

```diff
@@ -0,0 +1,44 @@
+package ru.sbt.mipt.oop.general;
+
+import ru.sbt.mipt.oop.commands.SensorCommand;
+import ru.sbt.mipt.oop.events.SensorEvent;
+import ru.sbt.mipt.oop.events.SensorEventType;
+import ru.sbt.mipt.oop.io.SmartHomeDataInput;
+import ru.sbt.mipt.oop.io.SmartHomeDataInputJSON;
+
+public class Application {
+
+    public static void main(String... args) throws Exception {
+        // get smartHome state from some source
+        SmartHomeDataInput smartHomeInput = new SmartHomeDataInputJSON("smart-home-1.js");
+        SmartHome smartHome = smartHomeInput.getData();
+        // setup command module
+        TotalCommander commander = new TotalCommander();
+        // start events handling cycle
+        SensorEvent event = getNextSensorEvent();
+        while (event != null) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%BF%D0%B5%D1%80%D0%B5%D0%BD%D0%B5%D1%81%D0%B8%D1%82%D0%B5-%D0%BB%D0%BE%D0%B3%D0%B8%D0%BA%D1%83-%D1%86%D0%B8%D0%BA%D0%BB%D0%B0-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B8-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%B2-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-%D0%BA%D0%BB%D0%B0%D1%81%D1%81